conf/init/nginx_conf_create.py

```diff
@@ -124,6 +124,21 @@ def generate_rate_limiting_config(config):
     )
 
 
+def generate_http_base_config(config):
+    """
+    Generates http base config from the app config.
+    """
+    config = config or {}
+    feature_proxy_protocol = config.get("FEATURE_PROXY_PROTOCOL", False)
+    trusted_proxy_ips = config.get("TRUSTED_PROXY_IPS", None)
```

Comment:
This would be a CIDR. Maybe rename to `TRUSTED_PROXY_CIDRS`

Reply:
Done. Although I think regular ip notation also works.